src/main/java/com/minecolonies/sounds/CitizenSounds.java

```diff
@@ -0,0 +1,81 @@
+package com.minecolonies.sounds;
+
+import com.minecolonies.util.SoundUtils;
+import net.minecraft.util.SoundEvent;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.World;
+
+/**
+ * Class used to store the basic citizen sounds.
+ */
+public final class CitizenSounds
+{
+    /**
+     * Chance to say a phrase.
+     */
+    private static final int PHRASE_CHANCE = 50;
+
+    /**
+     * Private constructor to hide the implicit public one.
+     */
+    private CitizenSounds()
+    {
+        /*
+         * Intentionally left empty.
+         */
+    }
+    /**
+     * The citizen sound events for the females.
+     */
+    public static final class Female
+    {
+        public static SoundEvent say;
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Make this "public static say" field final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1444)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make say a static final constant or non-public and provide accessors if needed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUndocumentedApi)